app/src/navigation/screens/PGPScreen.tsx

```diff
@@ -86,8 +86,18 @@ const PGPScreen: React.FC = () => {
     [generateKeyPair]
   );
 
+  const checkRequiredClaims = () => {
+    if (!emailClaimValue && !nameClaimValue) {
+      Alert.alert(
+        AlertTitle.Error,
+        "Email and Name claims must be set before a PGP/GPG Key can be generated."
+      );
+    }
+  }
+
   React.useEffect(() => {
     (async () => {
+      checkRequiredClaims;
```

Comment:
Needs brackets after statement to run function

Reply:
Nice catch